portfolio/src/main/webapp/travel.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title> Travel
+    </title>
```

Comment:
Nit: remove space before title and move to a single line.
```suggestion
    <title>Travel</title>
```